apps/spruce/src/constants/routes.ts

```diff
@@ -27,7 +27,7 @@ export enum PreferencesTabRoutes {
 
 export enum ImageTabRoutes {
   BuildInformation = "build-information",
-  EventLog = "event-log",
+  EventLog = "event-logs",
```

Comment:
nit. put this back because it could break someone's bookmark
```suggestion
  EventLog = "event-log",
```